modules/services/syncthing.nix

```diff
@@ -58,8 +68,8 @@ with lib;
           };
 
           Service = {
-            Environment = "PATH=${config.home.profileDirectory}/bin";
-            ExecStart = "${pkgs.qsyncthingtray}/bin/QSyncthingTray";
+            ExecStart =
+              "${config.services.syncthing.tray.package}/bin/${config.services.syncthing.tray.package.pname}";
```

Comment:
I imagine that there is no guarantee that the package `pname` attribute matches the executable's name, or indeed that there is a `pname` attribute at all.

Reply:
Yeah, it's the first heuristic I could thing of - do you know of any better ways?

Reply:
I think the only option is to simply ask the user to give the whole command 😕